engineyard/ey-cookbooks-stable-v5
# [FB-2259] Update-default-python2.7-Version2.7.17

Description of your patch
-------------
Updated the default `python27` version of the stack to `2.7.17`

Recommended Release Notes
-------------
Updated default `python2.7` version of the stack to `2.7.17`

Estimated risk
-------------
Low

Components involved
-------------
`security_updates` cookbook

Description of testing done
-------------
Created a `labrea` release with the changes and upgraded the current stack into the newest release, to see if the python version is upgraded.

QA Instructions
-------------
1. Check the python2.7 version change with a solo environment
2. Check the version check in a multiple instance environment
